pkgs/test/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Pass `--server-mode` to dart2js instead of `--categories=Server` to fix a
   warning about the flag deprecation.
+* Drop dependency on `pub_semver`.
```

Comment:
Worth a changelog entry? 🤷‍♂

Reply:
Some consumers might care. The direct consumers of `test_core` probably care more, but some might care about the drop from `test` as well.